incubator/hnc/pkg/reconcilers/hierarchical_namespace.go

```diff
@@ -80,32 +80,32 @@ func (r *HierarchicalNamespaceReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, r.writeInstance(ctx, log, inst)
 	}
 
-	// Get the owned child namespace instance. If it doesn't exist, initialize one.
-	cnsInst, err := r.getNamespace(ctx, nm)
+	// Get the subnamespace instance. If it doesn't exist, initialize one.
```

Comment:
Replace it with "Get the namespace instance of the subnamespace"? In case the `hierarchicalnamespace` resource will be renamed to `subnamespace` in the future.

Reply:
Good call. Changed to "Get the subnamespace."